User/Invite/index.ts

```diff
@@ -1,8 +1,9 @@
 import * as isoly from "isoly"
 import * as authly from "authly"
-import { Creatable as CreatableInvite } from "./Creatable"
+import { isly } from "isly"
+import { Creatable as InviteCreatable } from "./Creatable"
 
-export interface Invite extends CreatableInvite {
+export interface Invite extends Invite.Creatable {
 	issuer: string
```

Comment:
Isn't this an email?

Reply:
it is the name of the authentication server who issued the JWT